test/pricer_rule/utils.js

```diff
@@ -60,24 +62,30 @@ module.exports.createOrganization = async (accountProvider) => {
  * Creates and returns the tuple:
  *      (tokenRules, organizationAddress, token)
  */
-module.exports.createTokenEconomy = async (accountProvider) => {
+module.exports.createTokenEconomy = async (accountProvider, config = {}, eip20TokenConfig = {}) => {
   const {
     organization,
     organizationOwner,
     organizationWorker,
   } = await this.createOrganization(accountProvider);
 
-  const token = await this.createEIP20Token();
+  const eip20TokenDecimals = eip20TokenConfig.decimals || 18;
+  const createEIP20TokenConfig = {
+    symbol: eip20TokenConfig.symbol || 'OST',
+    name: eip20TokenConfig.name || 'Open Simple Token',
+    decimals: eip20TokenDecimals,
+  };
+  const token = await this.createEIP20Token(createEIP20TokenConfig);
 
   const tokenRules = await TokenRulesSpy.new();
 
-  const baseCurrencyCode = 'OST';
-
-  const conversionRate = 315;
+  const baseCurrencyCode = config.baseCurrencyCode || 'OST';
```

Comment:
Do we have the case for having customizable base currency code?

Reply:
PriceOracle contract can be deployed for different base currency also. e.g. UNP/USD 
UNP: UNSPALSH  
The config gives us that flexibility.

Reply:
In the test cases baseCurrencyCode = 'OST' only. I will update to 'OST' only.

Reply:
This is only for testing, it's not going to be used for general deployment.